### DISCUSSION TITLE
phone vibrating all he time

### DISCUSSION THREAD

**Discussion by andrearicci** [OpenTracksApp/OpenTracks#610] [Q&A]
The phone vibrates all the time, can I stop it? I could'nt find an option.
Thank you

**Comment by pstorch** [comment]
That sounds like an error. Can you force kill the app?
When does it start?

**Reply by andrearicci** [reply]
Yes I can force kill it. But it's the same when restarting it. 
The problem starts when I start tracking with the gps, or I start the gps while tracking. It's the sum of the two.

**Reply by pstorch** [reply]
So ist starts when the notification for recording starts. 
Hmm, no problem with other apps which vibrate? I guess you already tried a device restart?

**Reply by StefanKley** [reply]
I had the same problem today. Seems like a phone restart solved the issue for me.

**Reply by andrearicci** [reply]
> I had the same problem today. Seems like a phone restart solved the issue for me.

not for me. Rebooted but still there.
Btw I have a redmi note 4 with MIUI 11.0.2

**Reply by StefanKley** [reply]
And there it is again :-(
I checked indoors after the reboot and stopped tracking again. After andrearicci replied I checked again and let it track for a while. Could this be releated to getting GPS data? As soon as position data is displayed the vibration starts again.

I have a Xiaomi Redmi Note 3 with MIUI 10 (8.11.22 Beta) Android 6.0.1

**Reply by andrearicci** [reply]
yes it seems related to gps: after rebooting I started it indoor (no gps signal) and as soon as I went out and the signal was there the problem started again.

**Comment by dennisguse** [comment]
Could it be that the somehow the permissions for the GPS get lost and then the TrackRecordingService gets killed and restarted?
i would love to get a logcat to investigate this.

PS: Which version of OpenTracks did you use?
@StefanKley as well ;)

**Reply by andrearicci** [reply]
Hi, it's v3.14.1-75b8f3b9c / 3799.
How do I make a logcat? no idea!
Thank you

**Reply by StefanKley** [reply]
Hi, the version is v3.14.3-163af2dab / 3823 (via f-Droid installed today)
I will try to get you a logcat... (would appreciate advice on how to create it)

**Reply by StefanKley** [reply]
Okay I managed to create a log. From what I can see in there I'm not comfortable making that data public entirely. I will try to strip it down to (hopefully) relevant parts. I will drop the filtered log tomorrow.

I did notice something else. At this time of day my phone switches to dnd mode - so all sounds are disabled. The continuous vibration does not happen in that mode. Activating sound again recreates the issue...

**Reply by StefanKley** [reply]
Alright I would not sleep with that ticking in my head anyway :-D
Please see file attached (filtered the log for "dennisguse") - Tracking starts at 02-23 23:04:25.495 
A few seconds later with incoming GPS data the vibrating started...

[filtered_log.txt](https://github.com/OpenTracksApp/OpenTracks/files/6032139/filtered_log.txt)
Let me know if you need any more data ...

**Comment by dennisguse** [comment]
I created #611 as it sounds like a bug.
